docs/configuration/configuration.md

```diff
@@ -47,53 +47,44 @@ The other placeholders are specified separately.
 
 A valid example file can be found [here](/config/testdata/conf.good.yml).
 
-The global configuration specifies parameters that are valid in all other configuration
+### `<global>`
+
+The `global` section contains the global Prometheus server
```

Comment:
This doesn't seem accurate or helpful. It's a set of overall settings and defaults.

Reply:
That's what `global` means to me - overall.